src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/personalisation/Personalisation.java

```diff
@@ -834,8 +835,8 @@ public Template getTemplate(E notificationWrapper, Benefit benefit, Subscription
             benefit, notificationWrapper, notificationWrapper.getNewSscsCaseData().getCreatedInGapsFrom());
     }
 
-    public int calculateMaxDwpResponseDays(String benefitCode) {
-        if (benefitCode != null && benefitCode.equals("childSupport")) {
+    private int calculateMaxDwpResponseDays(BenefitType benefitType) {
+        if (benefitType.getCode() != null && benefitType.getCode().equals("childSupport")) {
```

Comment:
I know this was there before but should we be using an existing enum for the String `childSupport`? I can see there's an enum Benefit in sscs-common.